zebra-network/src/peer_set/initialize.rs

```diff
@@ -241,8 +242,8 @@ where
 async fn add_initial_peers<S>(
     config: Config,
     outbound_connector: S,
-    mut peerset_tx: mpsc::Sender<DiscoveredPeer>,
-    address_book_updater: mpsc::Sender<MetaAddrChange>,
+    mut peerset_tx: futures::channel::mpsc::Sender<DiscoveredPeer>,
+    address_book_updater: tokio::sync::mpsc::Sender<MetaAddrChange>,
```

Comment:
This is confusing,it is possible to just use tokio or std `mpsc` everywhere?

Reply:
I noticed that, and I tried to fix it, but it turned out to be pretty complicated.

I opened #3260 with the details.